### PR TITLE
1.6 dev attestations

### PR DESCRIPTION
**Pull request by jkowalleck**
fixes https://github.com/CycloneDX/specification/issues/192

task list for spec enhacement
- [x] schema: JSON
- [x] schema: XML  
- [x] schema: protobuff
- [x] examples/test cases

### CODE REVIEW

**Comment by jkowalleck** [comment]
created this PR to make an early review easier.

**Comment by jkowalleck** [comment]
same topic, but not part of this: https://github.com/CycloneDX/specification/compare/v1.5-dev-attestations?expand=1

File: schema/bom-1.6.schema.json [hunk 1]
```diff
@@ -3809,6 +4188,155 @@
...
+                "title": "Text",
+                "description": "The textual content of the requirement."
+              },
+              "descriptions": {
+                "type": "array",
+                "title": "Descriptions",
+                "description": "The supplemental text that provides additional guidance or context to the requirement, but is not directly part of the requirement.",
+                "items": { "type":  "string" }
+              },
+              "opencre": {
```
Comment (jkowalleck) [inline schema/bom-1.6.schema.json:4257]: hard to read: `opencre` for "OWASP OpenCRE"
maybe use `openCRE` or `open_cre`?

**Review by jkowalleck — COMMENTED** [review]
did a quick review.
see my thoughts below

File: schema/bom-1.6.schema.json (hunk 1 above)
Comment (stevespringett) [reply schema/bom-1.6.schema.json:4284]: I think we use camel case. I can change to `openCre` if that works?

**Comment by stevespringett** [comment]
@CycloneDX/core-team can I get a review of this prior to promotion to TC54 meeting on January 11th please?

**Review by jkowalleck — COMMENTED** [review]
did no entire review yet, will do early next year. 

still open: 
- https://github.com/CycloneDX/specification/pull/348#discussion_r1408285442

File: schema/bom-1.6.schema.json (hunk 1 above)
Comment (stevespringett) [reply schema/bom-1.6.schema.json]: opencre has been changed to openCre

**Pull request merged**